.env.example

```diff
@@ -9,3 +9,11 @@
 
 # DIRECT_URL is given to you if using Neon. If using another postgres instance, set it to the same value as DATABASE_URL
 # DIRECT_URL=postgres://postgres:123456@127.0.0.1:5432/dummy
+
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_c2FmZS1yYXZlbi02OC5jbGVyay5hY2NvdW50cy5kZXYk
```

Comment:
Even though this can be published, not sure if we want it to be like public in the Github... since people will be creating clerk accounts anyway for the secret key, I think it might be best to leave this blank